ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink

```diff
@@ -150,7 +170,7 @@ use_api_v1() {
 	PEER_PORT="$(extract_port "$PEER_HOSTPORT")"
 	PEER_PUBLICKEY="$(uci get wireguard.peer_"$PEER".publickey)"
 	PEER_LINKADDRESS="$(uci get wireguard.peer_"$PEER".link_address)"
-	PEER_ADDRESS="$(resolve_host "$PEER_HOST")"
+	PEER_ADDRESS="$(resolve_host "$PEER_HOST" | head -n 1)" # use the first resolved address of the peer
```

Comment:
The first one is not said to be the public ipv6.
I never experienced a situation where this was not the case, though I don't know if this is a must.

I am not suggesting to change anything though - as it used the first match before either

Reply:
What would you like to do? Do something like a `| grep -v "^fe80" | grep -v "^fd"`?